### PR TITLE
chore[CL]: collect fees readme update

### PR DESCRIPTION
**Pull request by czarcas7ic**
Closes: #XXX

## What is the purpose of the change

CL readme updates for collect fees (as well as overall README linting). This was extracted from the MsgCollectFees PR.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (yes / no)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (yes / no)
  - How is the feature or change documented? (not applicable   /   specification (`x/<module>/spec/`)  /  [Osmosis docs repo](https://github.com/osmosis-labs/docs)   /   not documented)

### CODE REVIEW

File: x/concentrated-liquidity/README.md [hunk 1]
```diff
@@ -147,128 +209,162 @@ $$price = (10^{4}) + (650010 * 0.01) = 16,500.10$$
...
+b) Having the front end round the tick's actual price to the nearest
+  human readable/desirable spot price
 
-One draw back of this implementation is the requirement to create many ticks that will likely never be used. For example, in order to create ticks at 10 cent increments for spot prices greater than _$10000_, a $exponentAtPriceOne$ value of -5 must be set, requiring us to traverse ticks 1-3600000 before reaching _$10,000_. This should simply be an inconvenience and should not present any valid DOS vector for the chain.
+One draw back of this implementation is the requirement to create many ticks
+that will likely never be used. For example, in order to create ticks at 10
+cent increments for spot prices greater than _$10000_, a $exponentAtPriceOne$
+value of -5 must be set, requiring us to traverse ticks 1-3600000 before
+reaching _$10,000_. This should simply be an inconvenience and should not
+present any valid DOS vector for the chain.
```
Comment (p0mvn) [inline x/concentrated-liquidity/README.md:292]: I don't think this makes much sense anymore after having a stable exponent at price one of -6

File: x/concentrated-liquidity/README.md [hunk 2]
```diff
@@ -147,128 +209,162 @@ $$price = (10^{4}) + (650010 * 0.01) = 16,500.10$$
...
-	FrozenUntil     time.Time
+ PoolId          uint64
+ Sender          string
+ LowerTick       int64
+ UpperTick       int64
+ TokenDesired0   types.Coin
+ TokenDesired1   types.Coin
+ TokenMinAmount0 github_com_cosmos_cosmos_sdk_types.Int
+ TokenMinAmount1 github_com_cosmos_cosmos_sdk_types.Int
+ FrozenUntil     time.Time
```
Comment (p0mvn) [inline x/concentrated-liquidity/README.md:328]: Can you update this message, please?

File: x/concentrated-liquidity/README.md [hunk 3]
```diff
@@ -375,44 +470,66 @@ discussed in the "Pool Creation" section.
...
 
-Given the parameters needed for calculating the tokens needed for creating a position for a given tick, the API in the keeper layer would look like the following:
+Given the parameters needed for calculating the tokens needed for creating a
+position for a given tick, the API in the keeper layer would look like the following:
 
 ```go
-ctx sdk.Context, poolId uint64, owner sdk.AccAddress, amount0Desired, amount1Desired, amount0Min, amount1Min sdk.Int, lowerTick, upperTick int64, frozenUntil time.Time
+ctx sdk.Context, poolId uint64, owner sdk.AccAddress, amount0Desired,
+amount1Desired, amount0Min, amount1Min sdk.Int,
+lowerTick, upperTick int64, frozenUntil time.Time
```
Comment (p0mvn) [inline x/concentrated-liquidity/README.md:532]: `frozenUntil` is obsoleted, better remove to avoid confusion

File: x/concentrated-liquidity/README.md [hunk 4]
```diff
@@ -766,56 +968,36 @@ Now, we update the swap state as follows:
...
-First, note that the fees are collected in tokens themselves rather than in units of liquidity.
-Thus, we need two accumulators for each token.
-
-TODO: explain the `accum` package and how it is used in CL.
-Reference these papers:
+> As a an LP, I want to earn fees on my capital so that I am incentivized to
+participate in active market making.
 
-- [Scalable Reward Distribution](https://uploads-ssl.webflow.com/5ad71ffeb79acc67c8bcdaba/5ad8d1193a40977462982470_scalable-reward-distribution-paper.pdf)
-- [F1 Fee Distribution](https://drops.dagstuhl.de/opus/volltexte/2020/11974/pdf/OASIcs-Tokenomics-2019-10.pdf)
```
Comment (p0mvn) [inline x/concentrated-liquidity/README.md:800]: Why is this removed? This should stay. This is our accumulator package

File: x/concentrated-liquidity/README.md [hunk 5]
```diff
@@ -766,56 +968,36 @@ Now, we update the swap state as follows:
...
+In Balancer-style pools, fees go directly back into the pool to benefit all LPs pro-rata.
+For concentrated liquidity pools, this approach is no longer feasible due to the
+non-fungible property of positions. As a result, we use a different accumulator-based
+mechanism for tracking and storing fees.
```
Comment (p0mvn) [inline x/concentrated-liquidity/README.md:997]: I think linking the papers removed above and our `osmoutils/accum` package itself here would be helpful

File: x/concentrated-liquidity/README.md [hunk 6]
```diff
@@ -824,176 +1006,123 @@ fields to be stored on various layers of state:
...
 
-Collecting fees is as simple as transferring the requested amount
-from the pool address to the position's owner.
+Once calculated, collecting fees is a straightforward process of transferring the
+calculated amount from the pool address to the position owner.
 
-After every epoch, the system iterates over all positions to call
-`collectFees` for each and auto-collects fees.
-
-Currently, there is no ability to collect manually to prevent spam.
```
Comment (p0mvn) [inline x/concentrated-liquidity/README.md:981]: Lol what? I know I wrote that 😆

**Review by p0mvn — APPROVED** [review]
Nice work🧹

File: x/concentrated-liquidity/README.md (hunk 4 above)
Comment (czarcas7ic) [reply x/concentrated-liquidity/README.md:800]: Added back!
